test/e2e/specs/updates/keyrotation/keyrotation.go

```diff
@@ -0,0 +1,147 @@
+package keyrotation
```

Comment:
I think this can go in specs/fakerp/keyrotation.go for now?

Reply:
alright